src/index.ts

```diff
@@ -36,6 +36,17 @@ export interface WithPromise {
   Promise?: PromiseConstructor;
 }
 
+export interface CallbackifyAllOptions {
+  /**
+   * Array of methods to ignore when callbackifying.
```

Comment:
I do notice the statement in line 19 that says "Array of methods to ignore when promisifying." If that is acceptable, I have no comments.  The Style guide doesn't really talk about this. I would love to see what the external reviewer says about such comments.